ansible/deploy.yml

```diff
@@ -204,7 +204,8 @@
       - name: Trigger start ae node
         command: "/bin/true"
         notify: "restart aeternity daemon"
```

Comment:
Why restart? We need start, otherwise running the playbook will always restart the node, this is not what we want